xapian-applications/omega/omegatest

```diff
@@ -1451,6 +1451,17 @@ printf '$hitlist{$list{$terms{T},|}/$field{tag1}|$field{tag2}|$field{tag3}}' > "
 testcase 'Tone/one|two|three' B=Tone
 testcase 'Tthree|Ttwo/one|two|three' B=Ttwo B=Tthree
 
+
+# Test failover script is run when the failover script is provided
+# and non-existent template file is passed in $include
+printf '$include{non_existant.template,$set{data,replacement_for_template}}$opt{data}' > "$TEST_TEMPLATE"
```

Comment:
Typo - it's "existent" not "existant".

This tests that the fallback code is evaluated, but doesn't test that the result of the evaluation is used.  It looks from the code like it will be, but if you'd written or a later programmer maintaining the code refactors it to

```
    eval(args[1], param);
```

(i.e. failed to assign to `value`) that would evaluate it but not use the value, but your tests here wouldn't catch that.

It would be better to test something like this:

```
$include{non_existent.template,foo$set{data,bar}}$opt{data}
```

That should give `foobar`.